### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by saschad**
Preview at http://copay.dueckers.de

### CODE REVIEW

**Comment by cmgustavo** [comment]
Hi @saschad, thanks a lot!! We are very pleased to receive this type of contributions to Copay

**Comment by cmgustavo** [comment]
One comment regarding this PR: To standardize the translation process, we use (for now) generic languages (not country specific), for example for french, we use *fr* instead *fr_FR*. 

Could you change the file property from de_DE to only *de*. Also to rename the file de_DE.po to *de.po*?

Thanks!

**Comment by saschad** [comment]
Hope this is better, sorry for the last unnecessary commit :)

**Comment by cmgustavo** [comment]
No problem. LGTM.

File: po/de.po [hunk 1]
```diff
@@ -0,0 +1,1112 @@
...
+msgid "Amount"
+msgstr "Betrag"
+
+#: public/views/walletHome.html
+msgid "Amount in"
+msgstr "Betrag in"
+
+#: public/views/preferencesLanguage.html
+msgid "Applying changes"
+msgstr "Änerungen anwenden"
```
Comment (greggzigler) [inline po/de.po]: should be "Änderungen"

File: po/de.po [hunk 2]
```diff
@@ -0,0 +1,1112 @@
...
+"Teilnahme mit spezifizierten erweiterten privaten Schlüssel nicht möglich"
+
+#: src/js/controllers/join.js
+msgid "Could not join wallet:"
+msgstr "Beteiligung am Wallet nicht möglich"
+
+#: src/js/controllers/walletHome.js
+msgid "Could not reject payment. Check you connection and try again"
+msgstr ""
+"Zahlung konnte nicht abgelehnt werden. Bitte Verbindung prüfen und erneut "
```
Comment (greggzigler) [inline po/de.po]: "konnte" vs "kann" inconsistent translation of "could" across these three messages

File: po/de.po [hunk 3]
```diff
@@ -0,0 +1,1112 @@
...
+msgid "Please enter the required fields"
+msgstr "Bitte die benötigten Felder ausfüllen"
+
+#: src/js/controllers/import.js
+msgid "Please, select your backup file"
+msgstr "Bitte die Sicherungsdatei wählen"
+
+#: src/js/controllers/index.js
+msgid "Portuguese"
+msgstr "Portuguese"
```
Comment (greggzigler) [inline po/de.po]: if you intended all language names to be represented in the target language, then i think this should be "português"

File: po/de.po [hunk 4]
```diff
@@ -0,0 +1,1112 @@
...
+msgid "Set up a password"
+msgstr "Passwort einrichten"
+
+#: public/views/preferencesEmail.html
+msgid ""
+"Setting up email notifications could weaken your privacy, if the wallet "
+"service provider is compromised. Information available to an attacker would "
+"include your wallet addresses and its balance, but no more."
+msgstr ""
+"Das einrichten einer E-Mail Benachrichtigung schwächt die Privatsphäre, wenn "
```
Comment (greggzigler) [inline po/de.po]: should be "Das Einrichten" with uppercase "E" ?

**Comment by matiu** [comment]
Ooops, sorry I merge without seen @greggzigler comments. @saschad Could you update it in other PR? Thanks and sorry.

**Comment by saschad** [comment]
done... thanks @greggzigler

**Pull request merged**